src/verilog/verilog_expr.h

```diff
@@ -14,9 +14,14 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <util/expr.h>
 
 /// The syntax for these A.B, where A is a module identifier and B
-/// is an identifier within that module.
+/// is an identifier within that module, given als symbol_exprt.
 class hierarchical_identifier_exprt : public binary_exprt
 {
+public:
+  const symbol_exprt &op1() const
```

Comment:
Maybe `op1` isn't a great choice here? How about `inner`? I think the comment in `verilog_typecheckt::convert_parameter_override` that explains what `op1` is confirms that this method should have a different name.